### DISCUSSION TITLE
Welcome to Practice-Exams-Platform Discussions!

### DISCUSSION THREAD

**Discussion by danieldanielecki** [Ditectrev/Practice-Exams-Platform#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by b3n3w** [comment]
Hi there 👋
I was searching for a platform to learn for some of my exams and stumbled across your repo. Wonder if I could use your raw data to develop a small swift app to practice offline on my phone. 

Thanks and really appreciate your work !

**Reply by danieldanielecki** [reply]
Yes, no problem :)
We've been planning to release a mobile app for that: https://github.com/Ditectrev/Practice-Exams-Platform/issues/16

It's a different technology tho, but if you submit a PR, for the time being, I could publish the Swift app under my development account and make it available for all our students (and link it to our page/repos/GitHub).

**Reply by danieldanielecki** [reply]
PS. On our [Discord](https://github.com/Ditectrev) there's a [Contribution section](https://discord.com/channels/1134107164257038369/1199411815403237377). Whether it'll be a PR, or a independent repo, you can post your idea there and maybe someone else would be willing to contribute as well. I'd personally would be very happy to keep your application under `ios` folder as a PR to this repo.
<img width="1177" alt="Screenshot 2024-05-22 at 11 50 24" src="https://github.com/Ditectrev/Practice-Exams-Platform/assets/12736263/67c58829-44e9-4275-9f2c-b48945ed8370">

**Comment by danieldanielecki** [comment]
as discussed on Discord, @b3n3w will create a separate repo for the iOS app.
